main.cpp

```diff
@@ -158,7 +36,7 @@ int main(int argc, char ** argv) //!OCLINT tests may be long
         assert(1 == 2);
 #endif
 
-        run_simulation_given_arguments(results);
+     //   run_simulation_given_sel_type(results);
```

Comment:
this is commented, so right now nothing happens, right? Was this left in by mistake?

Reply:
It should be taken out, but the problem stays